libraries/matrix/impl/src/main/kotlin/io/element/android/libraries/matrix/impl/room/RustRoomFactory.kt

```diff
@@ -70,30 +88,41 @@ class RustRoomFactory(
             )
         }
 
-    suspend fun create(roomId: RoomId): MatrixRoom? = withContext(createRoomDispatcher) {
-        var cachedPairOfRoom: Pair<RoomListItem, Room>?
+    suspend fun destroy() {
```

Comment:
Is this ever called? I couldn't find anything calling it.

Reply:
Oooops, it was, but I reverted some part and forgot about this one, thanks!